src/widgets/numeric-refinement-list/__tests__/numeric-refinement-list-test.js

```diff
@@ -130,7 +130,7 @@ describe('numericRefinementList()', () => {
           header: '',
           // eslint-disable-next-line max-len
           item:
-            '<label class="{{cssClasses.label}}">\n <input type="radio" class="{{cssClasses.radio}}" name="{{attributeName}}" {{#isRefined}}checked{{/isRefined}} />{{label}}\n</label>',
+            '<label class="{{cssClasses.label}}">\n  <input type="radio" class="{{cssClasses.radio}}" name="{{attributeName}}" {{#isRefined}}checked{{/isRefined}} />{{label}}\n</label>',
```

Comment:
Hmmm, this is not really expected isn't it?

Reply:
Yes it is, in fact due it's to the indentation in `defaultTemplate` ([link](https://github.com/algolia/instantsearch.js/blob/develop/src/widgets/numeric-refinement-list/defaultTemplates.js#L5)). You can see that at the begining of the line we have two spaces.

Reply:
But, this is not in the PR?

Reply:
I just need to change the test, it's probably due some internal change in `react-element-to-jsx-string` that is used by `expect-jsx`.